docs/faqs/faqs.md

```diff
@@ -0,0 +1,212 @@
+---
+id: faqs
+title: Frequently Asked Questions
+sidebar_label: FAQs
+slug: faqs/faqs
+---
+
+
+# FAQs
+
+These are some of the questions we are often asked about using Amplication
+
+
+<details>
+  <summary>What is Amplication?</summary>
+
+  Amplication is an open-source development tool. It helps you develop quality Node.js applications without spending time on repetitive coding tasks. Amplication auto-generates fully functional apps based on TypeScript and Node.js.
+
+</details>
+
+<details>
+  <summary>How does Amplication work?</summary>
+  
+  You just need to define your data models, roles, and permissions, and Amplication will generate a TypeScript Node.js application with everything you need already baked in. This includes fully functional REST API and GraphQL API for all your data models, authentication, role-based authorization, logging, and even an admin UI. 
+
+</details>
+
+
+<details>
+  <summary>Do I need to be an expert developer to work with Amplication?</summary>
+  
+ Amplication is a great tool for developers, whatever their experience.  Even though beginners can get up-and-running quickly with Amplication, you own the code and so can make full use of your code-writing talent to customize yor output and work your magic. 
+  
+</details>
+
+<details>
+  <summary>How do I get started with Amplication</summary>
+  
+ You can generate your first app in minutes. These instructions will walk you through the steps of creating an application, adding an entity, adding roles, and setting permissions on entities:
+
+
+ 
+ [Your First Application](https://docs.amplication.com/docs/first-app/)
+
+  
+</details>
+
+<details>
+  <summary>Is Amplication Free?</summary>
+  
+ The Community Edition (CE) is free and always will be. However, to ensure that we can continue to develop Amplication, the Enterprise Edition (EE) will provide enterprise functionality that may require payment. For more information about the CE and EE, read:
+
+ 
+ [Amplication’s licensing model](https://docs.amplication.com/docs/about/about/licensing/)
+
+  
+</details>
+
+<details>
+  <summary>What is the difference between the Community Edition and the Enterprise Edition?</summary>
+
+  
+  **Community Edition (CE)** – contains Amplication's core functionality. It is open-source and free to use for the whole community. The Amplication Community Edition continues to be licensed under the [Apache 2.0 license](https://github.com/amplication/amplication/blob/master/LICENSE).
+
+  **Enterprise Edition (EE)** - adds advanced operations, security, and commercially orientated features. The Enterprise Edition is built on top of the Community Edition and the additional functionality will be supplied as source-available. While users can view and download the code, ownership of the code remains with Amplication, meaning users cannot resell it. Use of the Amplication Enterprise Edition is subject to the Amplication [Terms of Service](https://amplication.com/terms)
```

Comment:
we need to clarify that we are talking about the platform code and not about the generated code...